src/main/java/com/logentries/net/AsyncLogger.java

```diff
@@ -666,7 +674,9 @@ public void run() {
 			} catch (InterruptedException e) {
 				// We got interrupted, stop
 				dbg( "Asynchronous socket writer interrupted");
```

Comment:
You could add this call under the if as well... once debug is disabled there is no point in calling it. ;)